### ISSUE TITLE
Default value Enum is missing quotes '

### ISSUE THREAD

**Issue by themmes** [digitallyinduced/ihp#528]
Generated tables with a column with Enum type can have a default value, however the default value is missing single quotes.

**Comment by themmes** [comment]
Hmm.. Now that I think of this, I remember seeing some ticket related to showing the user to add these quotes themselves...

**Comment by mpscholten** [comment]
Yeah we had some issue regarding this already. But I think this was only related to the Data Editor (not the schema designer). As long as it's still confusing we should improve this

**Comment by mpscholten** [comment]
Wrong ticket closed
